### PR TITLE
XSOAR Simple Dev to Prod Update - Oct 2023

### PR DESCRIPTION
**Pull request by beauchompers**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: N/A

## Description
Updated to make use of the Core Rest API integration, as the Demisto Rest API has been deprecated.

In addition, this pack was showing in the XSIAM Marketplace, updated the pack metadata as it's only for XSOAR.

Tested with 6.12 and 8.4 of XSOAR as well. 

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by ilappe** [comment]
Thanks @beauchompers for contributing.
please see the [validation errors](https://circleci.com/gh/demisto/content/727170)

**Comment by beauchompers** [comment]
> Thanks @beauchompers for contributing. please see the [validation errors](https://circleci.com/gh/demisto/content/727170)

@ilappe How exactly are we supposed to fix the field names for an existing pack?

**Comment by ilappe** [comment]
> > Thanks @beauchompers for contributing. please see the [validation errors](https://circleci.com/gh/demisto/content/727170)
> 
> @ilappe How exactly are we supposed to fix the field names for an existing pack?
you are right..
let wait for @melamedbn  review and then we will know ho to fix it for existing items

**Comment by melamedbn** [comment]
Hi @beauchompers,

I noticed the recent Release Notes include references to the Incident Type and Layout, but it seems there weren't any changes to these items. Typically, we'd only mention components in the RN when modifications have been made.

Could it be possible that the changes weren't saved properly, or might there be an oversight in documenting the updates? It's important for our users to have accurate RNs to understand the changes they're implementing.

If adjustments were indeed made, kindly detail them for clarity. If not, we might need to revise the RN to reflect only the items that have undergone changes to maintain transparency and accuracy.

Best regards,
Ben

**Comment by beauchompers** [comment]
@melamedbn @ilappe 

Updates have been made to the release notes, and I added those ignore entries to the pack ignore file.

File: Packs/XSOAR-SimpleDevToProd/ReleaseNotes/1_0_6.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- Updated descriptions to include reference to the Core REST API integration, as this can now be used in this pack. 
+
+##### JOB - XSOAR - Simple Dev to Prod
+
+- Updated descriptions to include reference to the Core REST API integration, as this can now be used in this pack. 
+
+#### Scripts
+
+##### CustomContentBundleWizardry
+- Updated the Docker image to: *demisto/python3:3.10.13.78960*.
```
Comment (ilappe) [inline Packs/XSOAR-SimpleDevToProd/ReleaseNotes/1_0_6.md:14]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
```

File: Packs/XSOAR-SimpleDevToProd/ReleaseNotes/1_0_6.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+##### IsDemistoRestAPIInstanceAvailable
+- Updated the Docker image to: *demisto/python3:3.10.13.78960*.
```
Comment (ilappe) [inline Packs/XSOAR-SimpleDevToProd/ReleaseNotes/1_0_6.md:17]:
```suggestion
- Updated the Docker image to: *demisto/python3:3.10.13.80014*.
```

**Comment by ilappe** [comment]
Hi @beauchompers 
thanks a lot for fixing the comments
there is still [3 fails ](https://github.com/demisto/content/actions/runs/6776484262)regarding the release note
thanks in advance!

**Review by melamedbn — APPROVED** [review]

**Comment by beauchompers** [comment]
> Hi @beauchompers thanks a lot for fixing the comments there is still [3 fails ](https://github.com/demisto/content/actions/runs/6776484262)regarding the release note thanks in advance!

The 3 fails all have to do with the Incident Type, Layout, etc that I REMOVED from the release notes because you asked me to.   I have re-included them.

**Comment by beauchompers** [comment]
@ilappe I don't understand what it's complaining about here, those are there?

[2023-11-08 19:30:21,207] [INFO] The files were found as invalid, the exact error message can be located above
Error: Did not find content items headers under "Incident Fields" - might be duo to missing "**" symbols in the header.
Please use "demisto-sdk update-release-notes -i Packs/XSOAR-SimpleDevToProd"
For more information, refer to the following documentation: https://xsoar.pan.dev/docs/documentation/release-notes
Error: Did not find content items headers under "Incident Types" - might be duo to missing "**" symbols in the header.
Please use "demisto-sdk update-release-notes -i Packs/XSOAR-SimpleDevToProd"
For more information, refer to the following documentation: https://xsoar.pan.dev/docs/documentation/release-notes

**Review by ilappe — APPROVED** [review]
Thanks @beauchompers 
 great work!

**Pull request merged**